_emails/newsletter-2021-03.html

```diff
@@ -0,0 +1,148 @@
+{% capture text %}
+During January we announced our plan to [decommission the FT Bower Registry and migrate to the public npm registry](https://origami.ft.com/blog/2021/01/18/deprecating-bower-and-origami-via-npm/#who-does-this-affect).
+
+Work has now started on migrating the Origami components onto npm, 8 out of 66 of our component are now ready to be migrated. We aim to have all components, including documentation/guidance for how to migrate your products, complete by the start of Q3.
+
+As a reminder, below is a copy of our planned timeline to decommission the FT Bower Registry.
+
+1st July 2021 - Origami Components moved from Bower to npm and guides to migrate are written
+
+1st July 2021 - Deprecate the FT Bower Registry
+
+1st July 2022 - Decommission the FT Bower Registry
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+
+<!-- Thing 2 -->
+{% include email/h3.html content="New “Introduction to Origami components”workshop dates announced" %}
+{% capture text %}
+This workshop is great for people who are new to Origami or want a refresher on how to build with Origami.
+
+The workshop contents has had a revamp, it has been rewritten to be better suited for a remote audience and includes xyz and abc
+
+The dates we plan to run the workshop on are:
+- 42nd of borktober (include link to the Google Calendar entry)
+- 57th of Coffeeuary (include link to the Google Calendar entry)
+{% endcapture %} {% include email/markdown.html content=text %}
+
+
+<!-- Thing 3 -->
+{% include email/h3.html content="Guidance on how to customise an Origami component" %}
+{% capture text %}
+We now have a page to help our users decide if they need to customise an Origami component, and if so, how to go about it in a safe, supported way which should reduce the chance of the customisation breaking in the future.
+
+Please [have a read through the page](https://origami.ft.com/docs/components/customisation/) and let us know of any feedback you may have or suggestions for other topics we could write guidance for.
+{% endcapture %} {% include email/markdown.html content=text %}
+
+
+
+<!-- Special thanks -->
+{% include email/h2.html content="Special Thanks" %}
+
+{% capture text %}
+We recently had an uptick in the amount of security researchers wanting to submit reports about potential issues on [polyfill.io](https://polyfill.io). We'd like to thank the Cyber Security team for communicating with the security researchers and reviewing their reports for us.
+{% endcapture %}
+{% include email/markdown.html content=text %}
+
+
+<!-- Broader update -->
+{% include email/h2.html content="Broader Update" %}
+
+{% capture text %}A digest list of other things that have happened this month:{% endcapture %}
+{% include email/markdown.html content=text %}
+
+{% capture text %}
+- [Financial-Times/origami-build-service](https://api.github.com/repos/Financial-Times/origami-build-service)
```

Comment:
```suggestion
- [Financial-Times/origami-build-service](https://api.github.com/repos/Financial-Times/origami-build-service)
```

These links are to `https://api.github.com/repos/` and should be to `https://github.com/`. Sorry I've been meaning to correct the script but instead have been lazily doing a find and replace for previous emails.